moodle/docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+# This Docker compose file is written to configure services required to deploy Moodle, e-learning platform,
+# on containers
+# Author: Zainab Alsaffar
+version: '3'
+volumes:
+  mariadb-data:
+    driver: local
+  moodle-data:
+    driver: local
+networks:
+    moodle-network:
+services:
+  mariadb:
+    image: 'mariadb:latest'
+    container_name: mariadb
+    environment:
+      MYSQL_ROOT_PASSWORD: YOUR_PASSWORD
+      MYSQL_DATABASE: moodle
+      MYSQL_USER: moodle
+      MYSQL_PASSWORD: YOUR_PASSWORD
+    volumes:
+      - 'mariadb-data:/var/lib/mysql'
+    networks:
+      - moodle-network
+  moodle:
+    image: 'bitnami/moodle:latest'
+    container_name: moodle
+    environment:
+      MOODLE_DATABASE_HOST: mariadb
+      MOODLE_DATABASE_USER: moodle
+      MOODLE_DATABASE_NAME: moodle
+      MOODLE_DATABASE_PASSWORD: YOUR_PASSWORD
+    ports:
+      - '80:8080'
+      - '443:8443'
```

Comment:
How is TLS configured? Should we have a central NGINX proxy configured to terminate TLS and reverse proxy our apps?

Reply:
@thatnerdjosh , That's a good question! Actually, I wasn't told to take care of this and I haven't seen this being done in other apps.